Formula/ydcv.rb

```diff
@@ -0,0 +1,17 @@
+class Ydcv < Formula
+  desc "YouDao Console Version - Simple wrapper for Youdao online translate (Chinese <-> English) service API, as an alternative to the StarDict Console Version(sdcv)."
+  homepage "https://github.com/felixonmars/ydcv"
+  url "https://github.com/felixonmars/ydcv/archive/0.4.tar.gz"
+  sha256 "2d9f6309bbf2d35c0c34c5ee945cf40769cc8201e6f374fa2a4f2d4b827fbdbb"
+
+  depends_on "python3"
```

Comment:
Github page states that Python 2.7 is supported

Reply:
The macOS default Python interpreter's version is 2.7.10. Should I add a `depends_on "python"` here?

Reply:
`depends_on :python if MacOS.version <= :snow_leopard`

Reply:
Done